### PR TITLE
[Tensilelite] Enable Silu

### PR DESCRIPTION
**Pull request by AndySu12**
Silu(x) = x * Sigmoid(x)

### CODE REVIEW

**Review by KKyang — CHANGES_REQUESTED** [review]
1. Please use rebase instead of merge.
2. The two commits have the same message, please fix it.

**Comment by AndySu12** [comment]
@KKyang Done, thanks.

File: tensilelite/Tensile/Activation.py [hunk 1]
```diff
@@ -765,6 +768,44 @@ def getDGeluModule(self, cDataType, vgprIn, vgprOut):
             raise RuntimeError("Unsupported data type %s."%cDataType.toDevice("HIP"))
         return module
 
+    def getSiluModule(self, cDataType, vgprIn, vgprOut):
```
Comment (KKyang) [inline tensilelite/Tensile/Activation.py:771]: we already have sigmoid module, any reasons why can't call sigmoid here then add a mul x?
def getSiluModule(...):
    module.addModuleAsFlatItem(self.getSigmoidModule(...))
    module.add(VMulF32(...))

**Review by KKyang — CHANGES_REQUESTED** [review]

File: tensilelite/Tensile/Activation.py (hunk 1 above)
Comment (AndySu12) [reply tensilelite/Tensile/Activation.py:771]: Good advice, I will use Sigmoid + mul. Initially I just use dedicated function, no further reason or consideration.

Comment (AndySu12) [reply tensilelite/Tensile/Activation.py:771]: Done.

**Review by KKyang — APPROVED** [review]

**Review by jichangjichang — APPROVED** [review]
lgtm

**Pull request merged**